test/test_jit.py

```diff
@@ -5154,26 +5098,9 @@ def fn():
         ast = torch.jit.frontend.get_jit_def(fn)
         self.assertExpected(str(ast))
 
-    @unittest.skipIf(PY2, "Requires python 3")
-    def test_python_frontend_py3(self):
-        def fn():
-            raise Exception("hello")
-        ast = torch.jit.frontend.get_jit_def(fn)
-        self.assertExpected(str(ast))
-
     def _make_scalar_vars(self, arr, dtype):
         return [torch.tensor(val, dtype=dtype) for val in arr]
 
-
-    @unittest.skipIf(PY2, "tuple printing in py2 is different than torchscript")
-    def test_string_print(self):
```

Comment:
This was deleted

Reply:
It's here https://github.com/pytorch/pytorch/pull/21489/files#diff-ff4f8670281cd1eb4e09329cc1dcb43bR50